application.conf

```diff
@@ -43,4 +43,12 @@ openam {
   testUser=""
   testUserPassword=""
   tokenUrl=""
+  deploymentUri = "replace_with_openam_deployment_uri"
+  realm = "/apollo"
+  username = "replace_with_openam_username"
+  password = "replace_with_openam_password"
+  authIndex {
+    type = "module"
+    value = "DataStore"
+  }
```

Comment:
I was stymied as to why my changes to logback.xml weren't affecting the logs you were writing. Turns out, I don't think our logging was *ever* set up correctly. I'm not certain of the best way to do this, and would love to hear your opinion, but what I think we need is:

1) add akka-slf4j as a libraryDependency to build.sbt. Need to find the right version of this that matches our scala version.
2) add something like this to application.conf:
```
akka {
  loggers = ["akka.event.slf4j.Slf4jLogger"]
  loglevel = "DEBUG"
  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
}
```

After all that, the akka system will honor logback.xml, and we should be good.